taworks/taform/views.py

```diff
@@ -235,7 +235,14 @@ def apply(request):
                     }
                 return render(request, 'taform/application.html', context)
             context = None
-            return HttpResponseRedirect('application_submitted.html')
+            studentID=request.POST['student_id']
+            courses_applied= models.Course.objects.filter(application__student__student_id=studentID).distinct()
```

Comment:
i would pull based on the PK of the student table.

1. if multiple students apply with the same ID this would it would show an aggregate of all courses?
2. filter by student preference != 0